shared/fs/banner/conflict-banner-container.tsx

```diff
@@ -18,7 +22,17 @@ type DispatchProps = {
   onStartResolving: () => void
 }
 const mapDispatchToProps = (dispatch, ownProps: OwnProps) => ({
-  onFeedback: () => {},
+  onFeedback: () =>
+    dispatch(
+      RouteTreeGen.createNavigateAppend({
+        path: [
+          {
+            props: {feedback: `Conflict Resolution failed in \`${ownProps.path}\`.\n`},
+            selected: SettingsConstants.feedbackTab,
+          },
+        ],
+      })
+    ),
```

Comment:
Hmm this doesn't seem to be used anywhere. Did you forget to add it to `actions` in the component? Also note that we don't have the feedback screen on desktop, so would probably need a `isMobile` guard there.

Reply:
We do have the feedback screen on desktop now, I worked on that so we could do this.

I'm confused by "doesn't seem to be used anywhere". It's listed as a routeProp in the feedback container, and passed there. It's included in the `feedbackAction` in the component.

Reply:
I meant to say `onFeedback` wasn't called in the component, but looks like I just missed it. My bad!

Sounds good; feedback screen in desktop sounds awesome!